tests/test_learning_k_means.py

```diff
@@ -0,0 +1,65 @@
+"""
+Unit tests for landmarks space.
+"""
+
+import geomstats
+import geomstats.backend as gs
+import geomstats.tests
+
+from geomstats.geometry import hyperbolic_space
+from geomstats.geometry import hypersphere
+from geomstats.learning.k_means import KMeans
+
+
+class TestLearningKMeans(geomstats.tests.TestCase):
+    _multiprocess_can_split_ = True
+
+    def setUp(self):
+        pass
+
+    def test_hyperbolic_space_k_means(self):
+        x = gs.rand(100, 2)
```

Comment:
maybe set: `dim = 2` at the beginning, and use it. It will be easier to adapt your code to other dimensions later ;) thanks!